packages/yswaps/scripts/multicall/ThreePoolCrvMulticall.ts

```diff
@@ -0,0 +1,103 @@
+import { BigNumber, constants, PopulatedTransaction, Signer, utils } from 'ethers';
+import { PendingTrade, TradeSetup } from '@scripts/types';
+import { IMulticall } from './IMulticall';
+import { ICurveFi, ICurveFi__factory, IERC20, IERC20__factory, IVault, IVault__factory } from '@typechained';
+import { impersonate } from '../utils';
+import zrx from '../libraries/zrx';
+import { mergeTransactions } from '@scripts/libraries/multicall';
+
+// 1) 3pool => [usdc|usdt|dai]
+// 2) [usdc|usdt|dai] => yvBOOST
+// 3) yvBOOST withdraw  => yveCRV
+
+export class ThreePoolCrvMulticall implements IMulticall {
+  private threeCrv: string = '0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490';
+  private yveCrv: string = '0xc5bDdf9843308380375a611c18B50Fb9341f502A';
+  private yvBoost: string = '0x9d409a0A012CFbA9B15F6D4B36Ac57A46966Ab9a';
+
+  match(trade: PendingTrade) {
+    return trade._strategy == '0xa48c616144FD4429b216A86388CAb0Eed990cE87' && trade._tokenIn == this.threeCrv && trade._tokenOut == this.yveCrv;
```

Comment:
avoid using addresses, set them as constants on top of file.
same on :25 :26 & :29